explainer-server/app/config/Config.scala

```diff
@@ -73,4 +73,6 @@ class Config @Inject() (conf: Configuration) extends AwsInstanceTags {
   val liveTableName = s"explain-maker-live-$stage"
   val workflowDataTableName = s"explain-maker-workflow-data-$stage"
 
+  val explainListPageSize = 20
```

Comment:
Constants in Scala typically start with a capital: http://docs.scala-lang.org/style/naming-conventions.html